lib/repl.js

```diff
@@ -222,6 +225,15 @@ function REPLServer(prompt,
       }
     }
 
+    if (!wrappedCmd && experimentalModules && /\bimport\b/.test(code)) {
+      const potentialWrappedCode = processTopLevelImport(code, self);
+      if (potentialWrappedCode !== null) {
+        code = potentialWrappedCode;
+        wrappedCmd = true;
+        awaitPromise = true;
+      }
+    }
```

Comment:
What if someone decides to split an `import` declaration over multiple lines? I think you may have to consider the logic for `Recoverable` errors (see existing code below). Specifically, you may need to throw your own `Recoverable` errors if the parsing fails because of an unfinished `import` declaration.

Reply:
the way that `defaultEval` is set up is such that i don't need to worry about what `Recoverable` is or how many lines someone enters something on (it really is a nice bit of code) 😄